### PR TITLE
Added more new features to release notes

### PR DESCRIPTION
**Pull request by JuanjiJG**
Hello team,

This new pull request adds more new features to the release notes.

Regards,
Juanjo

### CODE REVIEW

File: source/release-notes/release_3_7_0.rst [hunk 1]
```diff
@@ -22,6 +22,20 @@ With this improvement, it's now possible to merge configuration from several gro
 
 The new feature allows consulting all the agent's group information at *agent_groups* CLI and at the app with the RESTful API. You can learn more about this feature at the :ref:`multiple groups <grouping-agents>` documentation.
 
+New Wazuh module to monitor Microsoft Azure
+-------------------------------------------
+
+The new ``azure-logs`` module for Wazuh has the ability to obtain and read Azure logs through several service APIs. This helps to monitor all activity that occurs in the infrastructure, just by setting up the module to monitor the virtual machines that make up the infrastructure, which will send events to the Wazuh manager for analysis.
```
Comment (jesusgn90) [inline source/release-notes/release_3_7_0.rst]: Replace :

```
This helps to monitor all activity that occurs in the infrastructure, just by setting up the module to monitor the virtual machines that make up the infrastructure, which will send events to the Wazuh manager for analysis.
```
with:
```
This helps to monitor all the activity happening in the infrastructure, just by setting up the module to monitor the virtual machines that form the infrastructure, sending events to the Wazuh manager for analysis.
```

File: source/release-notes/release_3_7_0.rst [hunk 2]
```diff
@@ -22,6 +22,20 @@ With this improvement, it's now possible to merge configuration from several gro
...
+
+To learn more about this new module and how to configure it, check out the section :ref:`azure`.
+
+New Wazuh module to monitor Docker
+----------------------------------
+
+The new ``docker`` module for Wazuh makes easier to monitor and collect the activity events from your Docker containers such as starting, stopping or pausing.
```
Comment (jesusgn90) [inline source/release-notes/release_3_7_0.rst]: `stopping or pausing.` --> `stopping or pausing events.`

File: source/release-notes/release_3_7_0.rst (hunk 1 above)
Comment (JuanjiJG) [reply source/release-notes/release_3_7_0.rst]: Done!

File: source/release-notes/release_3_7_0.rst (hunk 2 above)
Comment (JuanjiJG) [reply source/release-notes/release_3_7_0.rst]: Done!

**Pull request merged**